flutter/lib/src/sentry_flutter_options.dart

```diff
@@ -181,6 +181,8 @@ class SentryFlutterOptions extends SentryOptions {
   bool autoAppStart = true;
 
   /// Automatically attaches a screenshot when capturing an error or exception.
+  ///
+  /// Requires adding the [SentryScreenshotWidget] to the widget tree.
```

Comment:
Maybe add a code snippet which shows that it should be above all other widgets. Otherwise it might be added arbitrarily and then it will capture just parts of the widget tree. This is especially important for the case where it is added below the material app, as it will then always just capture the initial route.

Reply:
Makes sense, similar to https://github.com/getsentry/sentry-docs/pull/5745